interfaces/vrep/DQ_SerialVrepRobot.m

```diff
@@ -0,0 +1,242 @@
+% CLASS DQ_SerialVrepRobot - Concrete class to interface with serial robots
+% in VREP.
+%
+% Usage:
+%       1) Drag-and-drop a serial robot to a VREP scene. For instance, a
+%       "my_robot" robot.
+%       2) Run
+%           >> vi = DQ_VrepInterface();
+%           >> vi.connect('127.0.0.1',19997);
+%           >> vrep_robot = DQ_SerialVrepRobot("my_robot", 7, "my_robot", vi);
+%           >> vi.start_simulation();
+%           >> vrep_robot.get_configuration_space_positions();
+%           >> pause(1);
+%           >> vi.stop_simulation();
+%           >> vi.disconnect();
+%       Note that the name of the robot should be EXACTLY the same as in
+%       VREP. For instance, if you drag-and-drop a second robot, its name
+%       will become "my_robot#0", a third robot, "my_robot#1", and so on.
+%
+%   DQ_SerialVrepRobot Methods:
+%       get_joint_names - Gets the joint names of the robot in the CoppeliaSim scene.
+%       set_configuration_space_positions - Sets the joint configurations of the robot in the CoppeliaSim scene.
+%       get_configuration_space_positions - Gets the joint configurations of the robot in the CoppeliaSim scene.
+%       set_target_configuration_space_positions - Sets the joint configurations of the robot in the CoppeliaSim scene as a target configuration for the joint controllers.
+%       get_configuration_space_velocities - Gets the joint velocities of the robot in the CoppeliaSim scene.
```

Comment:
`get_configuration_velocities()`

Reply:
Updated in [c3356ae](https://github.com/dqrobotics/matlab/pull/108/commits/c3356aeb51058ab0faf8ce7a60f92f8d4c711c6a).